pkg/minishift/update/update_test.go

```diff
@@ -110,3 +112,35 @@ func checkErr(t *testing.T, err error) {
 		t.Fatal(err)
 	}
 }
+
+func addMockedResponses(mockTransport *minitesting.MockRoundTripper) {
```

Comment:
TBH, I see both names as equivalent. But if you prefer addMockResponses, so be it.